_data/versions.yml

```diff
@@ -1,4 +1,74 @@
 v3.0:
+- title: v3.0.3
+  note: |
+    23 February 2018
+    
+    > **Important**: [Review the documentation](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/) carefully before attempting an upgrade. 
+    > You must upgrade to Calico v2.6.5 or later before you can upgrade to Calico v3.0.2.
```

Comment:
I think here, you may want "3.0.3" instead of "3.0.2"

but on the other hand, could they upgrade from 3.0.1 or 3.0.2 to 3.0.3?

Reply:
Yep.

And, 3.0.x is later than v2.6.5, so we're OK on that.